src/icrc2-minter/src/canister.rs

```diff
@@ -388,6 +389,27 @@ impl MinterCanister {
     pub fn idl() -> Idl {
         generate_idl!()
     }
+
+    /// Get mint orders for the given wallet address and token;
+    /// if `offset` and `count` are provided, returns a page of mint orders.
+    fn token_mint_orders(
+        wallet_address: H160,
+        src_token: Id256,
+        offset: Option<usize>,
+        count: Option<usize>,
+    ) -> Vec<(u32, SignedMintOrder)> {
+        get_operations_store()
+            .get_for_address(&wallet_address, None, None)
+            .into_iter()
+            .filter_map(|(operation_id, status)| {
+                status
+                    .get_signed_mint_order(Some(src_token))
+                    .map(|mint_order| (operation_id.nonce(), *mint_order))
+            })
+            .skip(offset.unwrap_or_default())
```

Comment:
After this change, the two new Optional parameters to the `get_for_address` fn are never used.

Reply:
#[query]
    /// Returns the list of operations for the given wallet address.
    /// Offset, if set, defines the starting index of the page,
    /// Count, if set, defines the number of elements in the page.
    pub fn get_operations_list(
        &self,
        wallet_address: H160,
        offset: Option<usize>,
        count: Option<usize>,
    ) -> Vec<(MinterOperationId, OperationState)> {
        get_operations_store().get_for_address(&wallet_address, offset, count)
    }